v2/event/event_unmarshal_test.go

```diff
@@ -829,6 +829,34 @@ func TestUnmarshalWithOrdering(t *testing.T) {
 				DataBase64:  true,
 			},
 		},
+		"base64 json encoded data v0.3 with data/data_base64 -> datacontenttype -> specversion -> datacontentencoding": {
```

Comment:
Can you do a similar one without a base64 data (e.g. like struct data)

Reply:
Done!